aten/src/ATen/native/Integration.cpp

```diff
@@ -53,6 +53,14 @@ Tensor do_cumulative_trapezoid(const Tensor& y, double dx, int64_t dim) {
     return (dx /2. * (left + right)).cumsum(dim);
 }
 
+DimVector add_padding(IntArrayRef curr_shape, int64_t target_size) {
+    DimVector new_sizes(target_size, 1);
+    for (int i = 0; i < curr_shape.size(); i++) {
```

Comment:
type `i` like `curr_shape.size()` (I prefer using decltype but whatever)

Reply:
If I am understanding correctly, I should use `decltype(curr_shape.size()) i = 0` in place of `int i = 0`?

If so, done.

Reply:
That's one way to do it. You can also do `auto i = decltype(curr_shape.size()){0}`, which is a common pattern, or find what the size type of curr_shape is and just type `i` directly as that